osu.Game/Screens/Play/ScreenWithBeatmapBackground.cs

```diff
@@ -10,6 +10,8 @@ public abstract class ScreenWithBeatmapBackground : OsuScreen
     {
         protected override BackgroundScreen CreateBackground() => new BackgroundScreenBeatmap(Beatmap.Value);
 
+        public override bool? AllowTrackAdjustments => true;
```

Comment:
I'm confused why this is here. Seems wrong?

Reply:
Should this be true only for `SongSelect`?

Reply:
Right, this is replacing the default specification at `OsuScreen`. I'd need to check on a per case basis comparing to master. The other case I can think of is potentially the multiplayer room lobby (where DT may take effect), but apart from that, only song select does sound correct.

Reply:
Yeah, `RoomSubScreen` is already set to `true`, will move this to `SongSelect` and remove `=> false`s where not needed then